### ISSUE TITLE
Graceful fail?

### ISSUE THREAD

**Issue by phtmgt** [codebuds33/webp-conversion-bundle#2]
At the moment I receive a critical error whenever the source does not exist, e.g.:

`An exception has been thrown during the rendering of a template ("The file "/Users/xxx/dev/projects/yyy/public/uploads/files/photo/5d2ea1ab56e45_42E6A0C6-2EB1-4637-AE6D-13B0E6211CF8.jpeg" does not exist").`

Have you thought of some kind of failover, e.g., serving a 1 pixel file instead?

**Comment by p11v** [comment]
What do you think about this?

```
/**
     * @param $html
     * @return string
     * @throws Exception
     */
    public function setWebpExtension($html)
    {
        $fullFilePath = "{$this->projectDir}/public{$html}";
        if (!file_exists($fullFilePath)) {
            return ' ';
        }

        $webPPath = WebPConverter::convertedWebPImagePath($fullFilePath);
        $options = [
            'saveFile' => true,
            'quality' => $this->quality
        ];

        if (!file_exists($webPPath)) {
            WebPConverter::createWebPImage($fullFilePath, $options);
        }
        return explode("{$this->projectDir}/public", $webPPath, 2)[1];
    }
```

It works for me and is simple enough.

**Comment by DennisCodeBuds** [comment]
Hi,

Thanks for your proposal. I have just released V0.1.2, by default it will do extactly what you asked.

Maybe add some bundle configuration to set a default placeholder image if the image can not be found.
